modules/zookeeper/k8s/0.1/facets.yaml

```
@@ -7,6 +7,89 @@ clouds:
 - gcp
 - azure
 - kubernetes
+intent: zookeeper
```

YAMLlint (1.37.1)

[error] duplication of key "intent" in mapping

(key-duplicates)